webpack/assets/javascripts/react_app/components/SettingUpdateModal/components/SettingForm/SettingForm.js

```diff
@@ -18,7 +18,9 @@ const SettingForm = ({
     let submitValues = { setting: values };
 
     if (setting && setting.settingsType === 'array') {
-      const splitValue = { value: values.value.split(',') };
+      const splitValue = {
+        value: values.value === '' ? [] : values.value.split(','),
+      };
       submitValues = { setting: splitValue };
```

Comment:
Btw, is this the same?
```suggestion
      const splitValue = values.value === '' ? [] : values.value.split(',');
      submitValues = { setting: { value: splitValue } };
```

Reply:
Yes, updated.